src/TSHWebServer.py

```diff
@@ -215,6 +222,31 @@ def set_route():
             )
         return "OK"
 
+    # Set player data
```

Comment:
The idea here is that this is one way of managing this data. But we can add other endpoints that mock up a `dict` to `emit` and it would do it accordingly, if needed. For example, "set-character-<player>" if we want an easy command for people that basically just stream 1v1 tournaments...?